build.sbt

```diff
@@ -735,7 +735,8 @@ lazy val complianceTests = projectMatrix
       ce3 ++ Seq(
         Dependencies.Http4s.circe.value,
         Dependencies.Http4s.client.value,
-        Dependencies.Weaver.cats.value % Test
+        Dependencies.Weaver.cats.value % Test,
+        "com.lihaoyi" %% "pprint" % "0.8.1"
```

Comment:
make a line for this in the `Dependencies` object. Also http://sca.la/percent

Reply:
lolll he's got a shorten url for it hahahaha